regen.sh

```diff
@@ -73,4 +73,6 @@ go run regen.go -go_out "$root/src" -pkg_prefix "$PKG" "$apidir" "$protodir"
 echo 1>&2 "Checking that the libraries build..."
 go build -v ./...
 
+gofmt -s -d -l -w . && goimports -w .
```

Comment:
`-d` and `-w`? Probably just `-w`?

```
$ gofmt -h
usage: gofmt [flags] [path ...]
  -cpuprofile string
    	write cpu profile to this file
  -d	display diffs instead of rewriting files
  -e	report all errors (not just the first 10 on different lines)
  -l	list files whose formatting differs from gofmt's
  -r string
    	rewrite rule (e.g., 'a[b:len(a)] -> a[b:]')
  -s	simplify code
  -w	write result to (source) file instead of stdout
```

Reply:
Done